lib/features/organisation/presentation/mock_slider_screen.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/organisation/cubit/organisation_cubit.dart';
+
+class MockSliderScreen extends StatelessWidget {
+  const MockSliderScreen({super.key});
+  static const String routeName = "/mock-slider";
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<OrganisationCubit, OrganisationState>(
+      builder: (context, state) {
+        return Scaffold(
+          body: Center(
+            child: Text(state.organisation.name),
+          ),
+        );
+      },
+    );
```

Comment:
Why are you using a BlocBuilder here? you're not building anything based on the organisation state

Reply:
Its a mocked page, i just wanted something quick on a different page. It can(and will) be replaced entirely by @pavelpavelko

Reply:
Still the PR shouldn't include this